### ISSUE TITLE
Questions regarding comparsions

### ISSUE THREAD

**Issue by sunny1401** [xiong-zhitong/GeoFM#1]
Hi - this is a very useful repo. Thank you for this. have two questions - 

1) In datasets I see multiple wrappers. If i download respective datasets, I can in theory use these repo and relevant datasets as is? 
2) Is there like a benchmark etc - where i can run my model and get scores across or I suppose I should pull the repo in and then run locally?

**Comment by xiong-zhitong** [comment]
Yes, currently you have to run locally.  Supporting other datasets is ongoing. Please try dofa+geobench first. You may need to tune the hyperparameters (lr, batch size) to get decent results.

**Comment by sunny1401** [comment]
thank you so much
